.gitbook/3-protocol-devs/1-dep-specs/2-data-deal.md

```diff
@@ -0,0 +1,117 @@
+# Data Deal
+
+- Status: Draft
+- Crated: 2023-01-03
+- Modified: 2023-01-03
+- Authors
+  - Youngjoon Lee <yjlee@medibloc.org>
+  - Gyuguen Jang <gyuguen.jang@medibloc.org>
+  - Hansol Lee <hansol@medibloc.org>
+  - Myongsik Gong <myongsik_gong@medibloc.org>
+  - Inchul Song <icsong@medibloc.org>
+
+
+## Synopsis
+
+This document defines a data deal which is a contract for data collecting and pay for provision in [DEP](../../1-users/3-data-exchange/0-about-dep.md).
+Data consumers create data deal by specifying what type of data and how much they want, and how much they are willing to pay for it.
+Data providers can provide their data to the deals that match the type of data they have.
+When a data provider provides data to the deal, the payout is distributed to the provider and oracle that validated the data.
+Also, data consumers can deactivate their data deal whenever they want and the remained budget would be refunded to the consumer's account.
+
+### Motivation
+
+Data consumers want different types of data, and even for the same type of data, they all differ in how much they want and how much they are willing to pay.
+Thus, the data deal was devised so that data consumers can determine the type of data they want, as much as they want, at a cost they want.
+
+### Definitions
+
+`Data Provider`, `Data Consumer` and `Oracle` are defined in [User Flow](./1-user-flow.md)
+
+## Technical Specification
+
+### Data Structure of Deal
```

Comment:
```suggestion
## Technical Specification

Data consumers should be able to post the information described below publicly, so that any data provider can see it. Also, data providers should be assured that a particular data consumer really posted the information.

### Data Structure of Deal
```